### DISCUSSION TITLE
Change mouse cursor speed

### DISCUSSION THREAD

**Discussion by jonahclarsen** [input-leap/input-leap#1917] [Feature Requests]
For whatever reason, when I'm moving my mouse from my mac to my Windows, the cursor ends up moving too slowly. Changing cursor speed settings on Windows (client) doesn't help, changing them on Mac (server) using LinearMouse does, but then it's too fast on mac.

It would be great to have a "mouse speed multiplier" setting, or if anyone has any other hacky solutions I'm open to that too.

**Comment by Foresteam** [comment]
Same issue. I have a laptop with scaling 2x, and the cursor feels really "lubed up"

**Comment by jonahclarsen** [comment]
Two years later and I'm still struggling with this lol

**Comment by jonahclarsen** [comment]
For anyone who's struggling with this too, I just switched to Deskflow, worth looking into as it's actively maintained and seems to have this feature.
